recipes/igl/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "igl" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/geometryprocessing/libigl-python-bindings/archive/{{ version }}.tar.gz
+  sha256: cc749e5742986b20a440672a2a426b2b72598b179b411163768584009f44d445
+
+build:
+  number: 0
+  skip: True  # [win]
+  script:
+    - "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+
+  host:
+    - python
+    - pip
+    - libblas
+    - libcblas
+    - numpy
+    - scipy
+
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - scipy
+
+test:
+  imports:
+    - igl
+
+about:
+  home: https://libigl.github.io
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE.GPL
+  summary: 'Simple python geometry processing library'
+  description: |
+    libigl is a simple python and C++ geometry processing library. We have a wide functionality including construction of sparse discrete differential geometry operators and finite-elements matrices such as the cotangent Laplacian and diagonalized mass matrix, and simple facet and edge-based topology data structures.
+  doc_url: https://libigl.github.io/tutorial/
+  dev_url: https://github.com/libigl/libigl/
+extra:
+  recipe-maintainers:
+    - teseoch
+    - fwilliams
+    - skoch9
+    - danielepanozzo
```

Comment:
@fwilliams, @skoch9, and @danielepanozzo are you OK to be maintainers here?

Reply:
yes

Reply:
yes

Reply:
Yes